Tests/Tests/SDWebCacheCategoriesTests.m

```diff
@@ -120,6 +120,16 @@ - (void)testUIButtonSetBackgroundImageWithURLNormalState {
                                }];
     [self waitForExpectationsWithCommonTimeout];
 }
+
+- (void)testUIButtonBackgroundImageCancelCurrentImageLoad {
```

Comment:
macOS's `NSButton` does not re-defined with macro. Because it does not support the state API like UIKit. This test should apply for `SD_UIKIT` only.

Reply:
Emm, Thanks